src/main/java/io/servertap/api/v1/models/Player.java

```diff
@@ -195,4 +197,8 @@ public Boolean isOp() {
     public void setOp(Boolean op) {
         this.op = op;
     }
+
+    public Whitelist toWhitelist(){
+        return new Whitelist().uuid(uuid).name(displayName);
```

Comment:
This is a really interesting pattern you've established here. Definitely awesome from a syntactic sugar POV but can you sell me on why it's better than `return new Whitelist(uuid, displayName);`? (obviously you'd have to change the constructor).

The only reason I ask is that it's different than all the other POJOs in this project, and I'm wondering if it would be "better" to eventually adopt this pattern, or stick with old faithful.

Reply:
This method was a remnant of a previous attempt to convert players to whitelist entries. I never ended up using it. I can remove it.

As to why I am using `return new Whitelist().uuid(uuid).name(displayName);`. I was following the pattern that `Player.java` and `Server.java` use as neither have a constructor and instead use methods such as: `public Server name(String name) {
        this.name = name;
        return this;
    }`

I haven't written java in a very long time and assumed that pattern was done out of nessesity. I can definitely make Whitelist use a standard constructor if that's what you want.

Reply:
All good for now, just something for us to think about in future. I will grab your code locally and test later on today and then we can look at getting this merged. Thanks again!